<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IA_PlantsvsZombies</class>
 <widget class="QMainWindow" name="IA_PlantsvsZombies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IA Plants vs Zombies</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>220</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>220</height>
         </size>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="QPlainTextEdit" name="plainTextEdit_2">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>190</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>190</height>
            </size>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Solución</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>250</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="Error">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>w:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="w">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>5</number>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>h:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="h">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>z:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="z">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Comenzar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Introduzca las filas donde saldrán los Zombis separados por espacio:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="minimumSize">
              <size>
               <width>580</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>580</width>
               <height>80</height>
              </size>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Número de generaciones:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="generacion">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <property name="minimum">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Número de individuos generados por generación:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>20</height>
                </size>
               </property>
               <property name="minimum">
                <number>15</number>
               </property>
               <property name="value">
                <number>15</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
